.env.example

```
@@ -1,8 +1,9 @@
 # Supabase Configuration
-# Get these from: https://supabase.com/dashboard → Your Project → Settings → API
-# Note: This project uses Supabase's new API keys system
-EXPO_PUBLIC_SUPABASE_URL=https://your-project-ref.supabase.co
-EXPO_PUBLIC_SUPABASE_PUBLISHABLE_KEY=sb_publishable_your_key_here
+STYTCH_PROJECT_ID=""
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)